### ISSUE TITLE
ModernWpf.dll cannot be referenced because it uses built-in support for WinRT

### ISSUE THREAD

**Issue by visusys** [Kinnara/ModernWpf#442]
Using 0.9.4, and when I run I get: 

```Error	NETSDK1149	ModernWpf.dll cannot be referenced because it uses built-in support for WinRT, which is no longer supported in .NET 5 and higher.  An updated version of the component supporting .NET 5 is needed. For more information, see https://aka.ms/netsdk1149```

I thought this was compatible with .NET5? Am I missing something?

**Comment by tgiachi** [comment]
Same here

**Comment by takuya-takeuchi** [comment]
https://aka.ms/netsdk1149 says

* Install Microsoft.Windows.CsWinRT package

And I changed

* Target OS version 10.0.17763 from 7.0 (default value on .NET 6.0)

````
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net6.0-windows10.0.17763.0</TargetFramework>
    <Nullable>enable</Nullable>
    <UseWPF>true</UseWPF>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.6.3" />
    <PackageReference Include="ModernWpfUI" Version="0.9.4" />
    <PackageReference Include="Prism.Unity" Version="8.1.97" />
    <PackageReference Include="Prism.Wpf" Version="8.1.97" />
  </ItemGroup>

</Project>
````

It works fine.

**Comment by takuya-takeuchi** [comment]
sorry, build is fine but it failed to run after modified App.xaml

**Comment by takuya-takeuchi** [comment]
OK.  This issue would be related to https://github.com/Kinnara/ModernWpf/issues/187.

So it works.

````
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net6.0-windows10.0.18362.0</TargetFramework>
    <Nullable>enable</Nullable>
    <UseWPF>true</UseWPF>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.6.3" />
    <PackageReference Include="ModernWpfUI" Version="0.9.4" />
    <PackageReference Include="Prism.Unity" Version="8.1.97" />
    <PackageReference Include="Prism.Wpf" Version="8.1.97" />
  </ItemGroup>

</Project>
````

![image](https://user-images.githubusercontent.com/6241854/166116782-9ca13165-c8f3-42aa-a908-83f20b158acd.png)

**Comment by Kinnara** [comment]
Resolved by #451.
